api/ANGLE_instanced_arrays.json

```diff
@@ -1,210 +1,165 @@
-        "support": null
-      },
-      "Edge": {
-        "support": null
-      },
-      "Edge Mobile": {
-        "support": null
-      },
-      "Firefox": {
-        "support": "33.0"
-      },
-      "Firefox for Android": {
-        "support": null
-      },
-      "Internet Explorer": {
-        "support": "11"
-      },
-      "IE Mobile": {
-        "support": null
-      },
-      "Opera": {
-        "support": null
-      },
-      "Opera Mobile": {
-        "support": null
-      },
-      "Safari": {
-        "support": null
-      },
-      "Safari Mobile": {
-        "support": null
-      },
-      "Servo": {
-        "support": null
-      },
-      "status": {
-        "experimental": false,
-        "standardized": true,
-        "stable": true,
-        "obsolete": false
-      }
-    }
-  },
-  "ANGLE_instanced_arrays.drawElementsInstancedANGLE": {
-    "Basic support": {
-      "Android": {
-        "support": null
-      },
-      "Chrome": {
-        "support": null
-      },
-      "Chrome for Android": {
-        "support": null
-      },
-      "Edge": {
-        "support": null
-      },
-      "Edge Mobile": {
-        "support": null
-      },
-      "Firefox": {
-        "support": "33.0"
-      },
-      "Firefox for Android": {
-        "support": null
-      },
-      "Internet Explorer": {
-        "support": "11"
-      },
-      "IE Mobile": {
-        "support": null
-      },
-      "Opera": {
-        "support": null
-      },
-      "Opera Mobile": {
-        "support": null
-      },
-      "Safari": {
-        "support": null
-      },
-      "Safari Mobile": {
-        "support": null
-      },
-      "Servo": {
-        "support": null
-      },
-      "status": {
-        "experimental": false,
-        "standardized": true,
-        "stable": true,
-        "obsolete": false
-      }
-    }
-  },
-  "ANGLE_instanced_arrays.vertexAttribDivisorANGLE": {
-    "Basic support": {
-      "Android": {
-        "support": null
-      },
-      "Chrome": {
-        "support": null
-      },
-      "Chrome for Android": {
-        "support": null
-      },
-      "Edge": {
-        "support": null
-      },
-      "Edge Mobile": {
-        "support": null
-      },
-      "Firefox": {
-        "support": "33.0"
-      },
-      "Firefox for Android": {
-        "support": null
-      },
-      "Internet Explorer": {
-        "support": "11"
-      },
-      "IE Mobile": {
-        "support": null
-      },
-      "Opera": {
-        "support": null
-      },
-      "Opera Mobile": {
-        "support": null
-      },
-      "Safari": {
-        "support": null
-      },
-      "Safari Mobile": {
-        "support": null
-      },
-      "Servo": {
-        "support": null
-      },
-      "status": {
-        "experimental": false,
-        "standardized": true,
-        "stable": true,
-        "obsolete": false
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "ANGLE_instanced_arrays": {
```

Comment:
Yeah, constants could be added later, if there is demand. They don't have their own MDN pages, so there will be no dedicated macro call for them like `{{compat("api.ANGLE_instanced_arrays.VERTEX_ATTRIB_ARRAY_DIVISOR_ANGLE")}}`.